Public/Invoke-DockerPull.ps1

```diff
@@ -0,0 +1,32 @@
+function Invoke-DockerPull {
+    [CmdletBinding()]
+    param (
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Registry,
+        [Parameter(mandatory=$true)]
+        [String]
+        $Image,
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Tag = 'latest',
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Digest = ''
```

Comment:
Docker pull (the command from the Docker tool) does not support specifying both tag AND digest, you must pick one.
So, I don't think we should support specifying both either. To prevent the caller from doing something silly, you can provide two sets of parameters, one with $Tag and one with $Digest.
See here for reference: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6#parametersetname-argument